### DISCUSSION TITLE
v2.6.3

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4011] [分享 / Share]
- 修复 `隐藏推荐直播` 失效. (#3993)
- 修复 `夜间模式` 下鼠标移至特定页排行榜时的显示问题. (#1052, PR #4001 by [Tinhone](https://github.com/Tinhone))

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.3'>v2.6.3</a>.</em>

**Comment by q1171597496** [comment]
DEV版本中开启软件时，无法正确显示用户栏
